src/Helpers/DotNotationHelper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+class DotNotationHelper
+{
+    public static function getValueUsingDotNotation(array $array, string $path): mixed
```

Comment:
```suggestion
    public static function getValueUsingDotNotation(array $array, string $path): ?array
```

`mixed` is too wide for this scope. Use nullable type.

Reply:
okey, I'm changing it now